index.html

```diff
@@ -3287,7 +3290,7 @@ <h2>Integrity of Related Resources</h2>
         <p>
 Any object in the [=verifiable credential=] that contains an `id`
 property MAY be annotated with integrity information by adding the
-`digestSRI` and `mediaType` properties.
+`digestSRI`, `digestMultibase`, and, optionally, the `mediaType` property.
```

Comment:
There may be a better rephrasing. As I understand it, `mediaType` may accompany either `digestSRI` or `digestMultibase`, whichever is present, and only one of the latter two is allowed.
```suggestion
`digestSRI` or `digestMultibase` property, either of which MAY be 
accompanied by the additionally optional `mediaType` property.
```